CHANGELOG.md

```diff
@@ -24,7 +24,7 @@ All notable changes to this project will be documented in this file. The format
 ### Changed
 - Callbacks are sent one by one to the same URL. In the previous implementation, each callback request created a new goroutine to send the callback, which could result in a potential DDoS of the callback receiver. The new approach sends callbacks to the same receiver in a serial manner. Note that URLs are not locked by the `callbacker` instance, so serial sends occur only within a single instance. In other words, the level of parallelism is determined by the number of `callbacker` instances.
 
-- The Callbacker service handles unsuccessful callback attempts by placing problematic receivers in quarantine, temporarily pausing callback delivery to them.
+- The Callbacker service handles unsuccessful callback attempts by delaying individial callbacks and retrying them again.
```

Comment:
Actually we should not change the changelog, but add a new entry to it like

"The Callbacker service handles unsuccessful callback attempts by delaying individial callbacks and retrying them again instead of putting an unsuccessful receiver in quarantine"

Then next version would be v1.3.12

Reply:
wait, rebase reverted that change? 🤔 ok, fixing again